File: projects/java/core/utils/src/main/java/utils/Timer.java
package utils;
public class Timer{
	private long startTime;
	private long endTime;
	public void start(){
		startTime = System.currentTimeMillis();
	}
	public void stop(){
		endTime = System.currentTimeMillis();
	}
	public long currentTime(){
		return System.currentTimeMillis()-startTime;
	}
	public long result(){
		return endTime - startTime;
	}
	public String toString(){
		String res = String.valueOf((double)result());
		return res;
	}
}	

File: .config/nvim/lua/timer.lua
timer = {startTime=0, finishTime=0, resultTime=0}
function timer:start()
	self.startTime = os.clock()	
end
function timer:finish()
	self.endTime = os.clock()
end
function timer:result()
	return self.endTime - self.startTime
end
return timer
